apiV1.yaml

```diff
@@ -91,6 +116,7 @@ paths:
           description: Resource not found
         '500':
           description: Internal error
+
   '/entry/publicCount':
```

Comment:
I think you forgot to add the documentation of route '/entr**ies**/publicCount' (with deprecation of route '/entry/publicCount')

Reply:
Yes, will do it tomorrow, thanks :)

Reply:
Memory failure on my side... again... :)